rust/src/treefile.rs

```diff
@@ -23,17 +23,22 @@ use crate::cxxrsutil::*;
 use anyhow::{anyhow, bail, Context, Result};
 use c_utf8::CUtf8Buf;
 use nix::unistd::{Gid, Uid};
+use once_cell::sync::Lazy;
 use openat_ext::OpenatDirExt;
 use ostree_ext::{glib, ostree};
+use regex::Regex;
+use serde::de::Error;
```

Comment:
This looks too specific to pull into the global namespace, maybe
`use serde::de::Error as SerDeError;` or so?

Reply:
Nested it to the specific function!